### PR TITLE
Podcast Player: Document template variables

### PR DESCRIPTION
**Pull request by obenland**
#### Changes proposed in this Pull Request:

* Adds phpcs exception for undefined variables for template files.
* Documents template variables.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* If you're an Automattician, include a shortlink to the p2 discussion with Jetpack Product here.

#### Testing instructions:

* Make sure the server-side rendering of the player still works. 

#### Proposed changelog entry for your changes:

* No changeling entry needed

### CODE REVIEW

**Review by marekhrabe — APPROVED** [review]
This is very elegant! I like the change around `echo` logic to avoid a need to silence a rule and the out-of-box solution with the phpcs variable analysis. I think it makes sense, since these files are used a bit differently from other codebase and those rules don't make sense there.

File: extensions/blocks/podcast-player/podcast-player.php [hunk 1]
```diff
@@ -139,6 +139,7 @@ class="<?php echo esc_attr( $player_classes_name ); ?>"
 					$player_props,
 					array(
 						'primary_colors' => $primary_colors,
+						'player_id'      => $player_data['playerId'],
```
Comment (retrofox) [inline extensions/blocks/podcast-player/podcast-player.php:142]: are we duplicating this value in order to do tests pass?

**Comment by retrofox** [comment]
Nice improvement. Thanks :-)

**Comment by getdave** [comment]
I learned a tonne here. Thank you @obenland!

**Pull request merged**